app/models/katello/erratum.rb

```diff
@@ -61,12 +61,16 @@ def self.with_identifiers(ids)
       where("#{self.table_name}.id in (?) or #{self.table_name}.uuid in (?) or #{self.table_name}.errata_id in (?)", id_integers, ids, ids)
     end
 
-    def hosts_applicable
-      self.content_facets_applicable.joins(:host)
+    def hosts_applicable(org_id)
```

Comment:
If organization_id isn't supplied params[:organization_id] will be nil anyway, but we can do this as well.